people-ops/hiring.html.md

```diff
@@ -44,23 +52,44 @@ experience with software development and with [the tools we use](https://nebulab
 approach to work and why you are interested in Nebulab.
 
 Someone from our team will read your letter and, if they think you might be a good candidate, they
-will write to you to find the best time for an [informal chat](#informal-chat).
+will send you a link to the [aptitude questionnaire](#aptitude-questionnaire) which is the first
+step in the hiring process.
+
+### Aptitude questionnaire
+
+The aptitude questionnaire is a way for us to get to know you better and to understand how you think,
+act and communicate. The test will be mainly focused on your soft skills but there are some basic
+technical questions as well. The questionnaire will be sent to you as a Google form, which you can
+complete at your own pace. The only thing we ask is that you tell us when we can expect your answers,
+so we can do a better job of coordinating our hiring efforts.
+
+The fundamental requirement in answering the questionnaire is sincerity: write in accordance with
+your values and the way you work, not what you think we would like to hear from you. The purpose of
+the questionnaire is to understand whether you're a good cultural and technical fit, so it is in
+everyone's interest that the answers reflect the way you really work and handle problems as much as
+possible.
```

Comment:
I think we can remove "as much as possible" here. We want the questionnaire to fully reflect the person.